mandatory/3-function-output.js

```diff
@@ -1,16 +1,25 @@
+/** @format */
+
 // Add comments to explain what this function does. You're meant to use Google!
+
+//This getRandomNumber() gives a random number between 0 and 10
 function getRandomNumber() {
-  return Math.random() * 10;
+	return Math.random() * 10; //Math.random will return a number between 0 and 1
```

Comment:
Is Math.random inclusive of 1? And consequently is your method inclusive of 10?  Check the definition here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

Reply:
learned now that this will not be inclusive of 1 or 10 so we have to add 1 to make these numbers inclusive, thanks really helped me.